boa/src/syntax/ast/node/declaration/function_decl/mod.rs

```diff
@@ -89,14 +89,12 @@ impl Executable for FunctionDecl {
     fn run(&self, context: &mut Context) -> Result<Value> {
         let _timer = BoaProfiler::global().start_event("FunctionDecl", "exec");
         let val = context.create_function(
+            self.name(),
             self.parameters().to_vec(),
             self.body().to_vec(),
-            FunctionFlags::CALLABLE | FunctionFlags::CONSTRUCTABLE,
+            FunctionFlags::CONSTRUCTABLE,
```

Comment:
was CALLABLE never needed here?

Reply:
This PR also fixes #1441, That's why `FunctionFlags::CALLABLE` was removed, since it's implied.